nevergrad/optimization/oneshot.py

```diff
@@ -383,6 +394,13 @@ def __init__(
 MetaTuneRecentering = SamplingSearch(
     cauchy=False, autorescale="autotune", sampler="Hammersley", scrambled=True
 ).set_name("MetaTuneRecentering", register=True)
+HAvgMetaTuneRecentering = SamplingSearch(
```

Comment:
if H stands for Hull, maybe write Hull, that would be clearer (but not consistent with other legacy names) :s